code/src/Components/LikedMessage.css

```diff
@@ -0,0 +1,40 @@
+.heart-image{
+  display: flex;
+  justify-content: space-around;
+  margin-bottom: 2.5px;
+}
+
+.liked{
+  /* -webkit-appearance: none; */
+  background: rgb(250, 168, 168);
+  border-radius: 50%;
+  border: none;
+  height: 35px;
+  width: 35px;
+  margin: 4px 10px 10px 12px;
+}
+
+.not-liked{
```

Comment:
Instead of repeating the code for .liked and .not-liked (since the only difference is the background colour), you could style just the button element and then styled .not-liked to the required bg colour.

Reply:
I tried doing that, but it did not work as I intended - so this was the solution for now :)